Source/SwiftLintFramework/Rules/Lint/BlanketDisableCommandRule.swift

```diff
@@ -0,0 +1,143 @@
+import SwiftSyntax
+
+struct BlanketDisableCommandRule: ConfigurationProviderRule {
+    typealias ConfigurationType = BlanketDisableCommandConfiguration
+
+    var configuration = BlanketDisableCommandConfiguration()
+
+    init() {}
+
+    static let description = RuleDescription(
+        identifier: "blanket_disable_command",
+        name: "Blanket Disable Command",
+        description: "swiftlint:disable commands should be re-enabled before the end of the file",
+        kind: .lint,
+        nonTriggeringExamples: [
+            Example("""
+            // swiftlint:disable unused_import
+            // swiftlint:enable unused_import
+            """),
+            Example("""
+            // swiftlint:disable unused_import unused_declaration
+            // swiftlint:enable unused_import
+            // swiftlint:enable unused_declaration
+            """),
+            Example("// swiftlint:disable:this unused_import"),
+            Example("// swiftlint:disable:next unused_import"),
+            Example("// swiftlint:disable:previous unused_import")
+        ],
+        triggeringExamples: [
+            Example("// ↓swiftlint:disable unused_import"),
+            Example("""
+            // ↓swiftlint:disable unused_import unused_declaration
+            // swiftlint:enable unused_import
+            """),
+            Example("""
+            // swiftlint:disable unused_import
+            // ↓swiftlint:disable unused_import
+            // swiftlint:enable unused_import
+            """),
+            Example("""
+            // ↓swiftlint:enable unused_import
+            """)
+        ].skipWrappingInCommentTests().skipDisableCommandTests()
```

Comment:
Can the `.skipWrappingInCommentTests()` be removed with your fixes in #4799?

Reply:
Possibly - I'll give that a go. Re #4799, not sure if you read my recent comment there, but I think just removing support for commands in multiline comments completely might be the way to go.

Reply:
Yep, so the fixes I have in #4799 do not resolve `.skipWrappingInCommentTests` generally, partly because the general issue with commands in block comments (only the first one will be recognised) still remain, and because commands should still work in comments.

Reply:
Got it. 👍